cli/types/index.d.ts

```diff
@@ -2960,6 +3092,14 @@ declare namespace Cypress {
     expiry?: string
   }
 
+  interface EnqueuedCommand {
+    name: string
+    args: any[]

```

Comment:
I think `any` is all that we can guarantee. There could be a lot of work to create an interface of internal mapping of named commands like `select` with argument shapes. I don't know if that is worthwhile right now. It would make more sense if command implementations are written in TypeScript. This at least won't give any errors. I consider these events pretty advanced and will probably require an understanding of the source code anyways. `any` could always be cast to a more specific type in that case.

Reply:
yes